### PR TITLE
Alerter ❗️

### PR DESCRIPTION
**Pull request by y-lohse**
Following up on : #263, I've essentially copied over the Alerter component from Photos, updated it and added some documentation. [Here's the preview!](http://yannick-lohse.fr/cozy-ui/react/#alerter)

### CODE REVIEW

File: stylus/components/button.styl [hunk 1]
```diff
@@ -207,8 +206,10 @@ $button-alert--info
 
     &:active
     &:hover
+    &:not([disabled]):not([aria-disabled=true]):hover
```
Comment (y-lohse) [inline stylus/components/button.styl]: This is not cool, but there's another selector like that earlier in the file that forces a blue background on all buttons. Any recommendation @GoOz ?

**Comment by y-lohse** [comment]
[(1) Ajouter une modale de confirmation lors de la suppression d'un contact](https://trello.com/c/pnIW5A5u/823-1-ajouter-une-modale-de-confirmation-lors-de-la-suppression-dun-contact)

File: react/Alerter/index.jsx [hunk 2]
```diff
@@ -1,142 +1,203 @@
...
+import PropTypes from 'prop-types'
+import Portal from 'preact-portal'
 
 import styles from './styles.styl'
+import classNames from 'classnames'
 
-const ALERT_SHOW = 'ALERT_SHOW'
-const ALERT_DISMISS = 'ALERT_DISMISS'
-const ALERT_CLEAR = 'ALERT_CLEAR'
+const createStore = () => {
```
Comment (enguerran) [inline react/Alerter/index.jsx]: Can't we use redux and react-redux? Or something like that https://github.com/developit/state-machine-component?

**Comment by ptbrowne** [comment]
Great, thanks yannick ! I wondered why the transition was a bit choppy and saw that top/bottom are used for the transition. Can you change them to a transform ? It will be much smoother.

https://csstriggers.com/

> Changing top alters the geometry of the element. That means that it may affect the position or size of other elements on the page, both of which require the browser to perform layout operations.

Once those layout operations have completed any damaged pixels will need to be painted and the page must then be composited together.

> Changing transform does not trigger any geometry changes or painting, which is very good. This means that the operation can likely be carried out by the compositor thread with the help of the GPU.

**Pull request merged**